packages/vulnerabilities/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redhat-cloud-services/vulnerabilities-client",
-  "version": "1.2.13",
+  "version": "1.2.14",
```

Comment:
Last thing before release. We will have to make sure, all the tags, releases and actual npm package versions are in sync. For example this version does not exist on npm:

https://www.npmjs.com/package/@redhat-cloud-services/vulnerabilities-client?activeTab=versions